File: c#/1012.cs
using System;
using System.Globalization;

class URI {

    static void Main(string[] args) {

        string input = Console.ReadLine();
        string[] parts = input.Split(' ');
        
        double a = double.Parse(parts[0], CultureInfo.InvariantCulture);
        double b = double.Parse(parts[1], CultureInfo.InvariantCulture);
        double c = double.Parse(parts[2], CultureInfo.InvariantCulture);

        double triangulo = a * c / 2;
        double circulo = 3.14159 * c * c;
        double trapezio = (a + b) * c / 2;
        double quadrado = b * b;
        double retangulo = a * b;

        Console.WriteLine($"TRIANGULO: {triangulo.ToString("F3", CultureInfo.InvariantCulture)}");
        Console.WriteLine($"CIRCULO: {circulo.ToString("F3", CultureInfo.InvariantCulture)}");
        Console.WriteLine($"TRAPEZIO: {trapezio.ToString("F3", CultureInfo.InvariantCulture)}");
        Console.WriteLine($"QUADRADO: {quadrado.ToString("F3", CultureInfo.InvariantCulture)}");
        Console.WriteLine($"RETANGULO: {retangulo.ToString("F3", CultureInfo.InvariantCulture)}");

    }      
}

File: c++/1012.cpp
#include <stdio.h>
#include <math.h>

int main() {
 
    double A, B, C;
    char line[50];
    double tri, cir, tra, qua, ret;

    fgets(line, sizeof(line), stdin);
    sscanf(line, "%lf %lf %lf", &A, &B, &C);

    tri = A * C / 2.0;
    cir = 3.14159 * pow(C, 2);
    tra = (A + B) * C / 2.0;
    qua = pow(B, 2);
    ret = A * B;

    printf("TRIANGULO: %.3lf\n", tri);
    printf("CIRCULO: %.3lf\n", cir);
    printf("TRAPEZIO: %.3lf\n", tra);
    printf("QUADRADO: %.3lf\n", qua);
    printf("RETANGULO: %.3lf\n", ret);
 
    return 0;
}